package.json

```diff
@@ -80,9 +80,10 @@
 		"postcss-preset-env": "^6.7.0",
 		"react-test-renderer": "^18.2.0",
 		"rimraf": "^5.0.5",
+		"sass": "^1.75.0",
 		"simple-git-hooks": "^2.7.0",
 		"stylelint": "^13.8.0",
-		"typescript": "^5.3.2",
+		"typescript": "^5.4.5",
```

Comment:
Upgrading TypeScript was necessary to fix a bug with how TypeScript handles the `target` compiler option.